app/src/main/java/cm/aptoide/pt/editorial/EditorialManager.java

```diff
@@ -89,7 +89,7 @@ private void setupDownloadEvents(Download download, String packageName, long app
     editorialAnalytics.setupDownloadEvents(download, campaignId, abTestGroup,
         AnalyticsManager.Action.CLICK, offerResponseStatus);
     installAnalytics.installStarted(download.getPackageName(), download.getVersionCode(),
-        AnalyticsManager.Action.INSTALL, AppContext.EDITORIAL,
```

Comment:
Will this change the context parameter name in the BI? We cannot change this values or we will loose data in the bi dashboards.